WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift

```diff
@@ -358,17 +358,25 @@ private extension OrdersRootViewController {
             return
         }
 
-        let coordinatingController = AddOrderCoordinator(siteID: siteID,
-                                                         sourceBarButtonItem: sender,
-                                                         sourceNavigationController: navigationController)
-        coordinatingController.onOrderCreated = { [weak self] order in
+        let viewModel = EditableOrderViewModel(siteID: siteID)
+        viewModel.onFinished = { [weak self] order in
             guard let self = self else { return }
 
             self.dismiss(animated: true) {
                 self.navigateToOrderDetail(order)
             }
         }
-        coordinatingController.start()
+
+        let viewController = OrderFormHostingController(viewModel: viewModel)
+        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.splitViewInOrdersTab) {
+            let newOrderNC = WooNavigationController(rootViewController: viewController)
+            navigationController.present(newOrderNC, animated: true)
```

Comment:
Nit: Should we rename this variable for clarity? Something like `newOrderNavigationController` instead?

Reply:
Great idea! Changed in 3adae6bec14662a7a3a9a21a1f9fc724974c0703